### PR TITLE
pythonPackages.blinker: execute tests

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by fabaff — APPROVED** [review]
Diff looks good to me. At least the 50 % I checked.

File: pkgs/development/python-modules/blinker/default.nix [hunk 1]
```diff
@@ -9,6 +9,9 @@ buildPythonPackage rec {
     sha256 = "1dpq0vb01p36jjwbhhd08ylvrnyvcc82yxx3mwjx6awrycjyw6j7";
   };
 
+  propagatedBuildInputs = [ nose pytestCheckHook ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/blinker/default.nix:12]:
```suggestion
  checkInputs = [ nose pytestCheckHook ];
```

🤦🏼

Comment (fabaff) [reply pkgs/development/python-modules/blinker/default.nix:12]: Yes, would be better to place those two there :wink:

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 113693` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages built:
</summary>
<ul>
<li>python37Packages.blinker</li>
<li>python38Packages.blinker</li>
<li>python39Packages.blinker</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

<details>
python37Packages.blinker:
<details>


Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/blinker/default.nix:19:5:

```
   |
19 |     maintainers = with maintainers; [ ];
   |     ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md
</details>
python38Packages.blinker:
<details>


Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/blinker/default.nix:19:5:

```
   |
19 |     maintainers = with maintainers; [ ];
   |     ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md
</details>
python39Packages.blinker:
<details>


Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/blinker/default.nix:19:5:

```
   |
19 |     maintainers = with maintainers; [ ];
   |     ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md
</details>
</details>

**Pull request merged**